Pages/homePage.js

```diff
@@ -0,0 +1,27 @@
+class HomePage {
```

Comment:
remove "var dataJS = require("../Data/data");"  from page objects where it is unused